app/components/Channels/ChannelSummaryList.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Box } from 'rebass'
+import { ChannelSummaryListItem } from 'components/Channels'
+
+const ChannelSummaryList = ({
+  channels,
+  cryptoCurrency,
+  cryptoCurrencies,
+  setCryptoCurrency,
+  showChannelDetail,
+  ...rest
+}) => (
+  <Box as="article" {...rest}>
+    {channels.map((channelObj, index) => {
+      const channel = Object.prototype.hasOwnProperty.call(channelObj, 'channel')
+        ? channelObj.channel
+        : channelObj
+
+      return (
+        <ChannelSummaryListItem
+          key={index}
```

Comment:
Can we use something other than index? `chan_id` looks good for this purpose. Using indexes as a key is generally an antipattern

Reply:
`chan_id` isn't always set (not set for pending nodes for example). I've created a new key which is a combination of a few different things which I think should be guaranteed to be unique.